pkg/kubectl/cmd/config/config.go

```diff
@@ -429,6 +430,10 @@ func getConfigFromFileOrDie(filename string) *clientcmdapi.Config {
 		return clientcmdapi.NewConfig()
 	}
 
+	if config.Clusters == nil {
+		config.Clusters = map[string]*clientcmdapi.Cluster{}
```

Comment:
@smarterclayton, you just did some work in this area... does this look like what you would expect here?